docs/pages/_bundles/2CamelConfig.md

```diff
@@ -7,16 +7,84 @@ permalink: bundles/camelconfig
 # CamelConfig
 
 ## The goal of the bundle
-CamelConfig is a bundle designed to parse the Apache Camel xml flows and translate this into Symfony services.
+CamelConfig is a bundle designed to parse the Apache Camel XML flows and translate this **into Symfony services**.
 
 ## Main features
+
+* Easy to use and implement.
+* Route versioning.
+* Full support for complex routing, like *multicasts* or *recipient lists*.
+* Clear route definition, based **on Apache's specification**.
+* Fully compatible **with Symfony 2.8**. (Compatibility with 3.4 is part of [the roadmap](smartesb-skeleton/roadmap))
+* Battle tested.
+
 ## Installation
+Installation is as easy as installing a new bundle in your Symfony application. First start by running `composer`:
+
+`$ composer require smartbox/camel-config-bundle`
+
+Then you need to **enable the bundle** in your kernel.
+
+```php
+// app/AppKernel.php
+
+class AppKernel extends Kernel
+{
+    public function registerBundles()
+    {
+        $bundles = [(
+            new Smartbox\Integration\CamelConfigBundle\SmartboxIntegrationCamelConfigBundle(),
+            (...)
+```
+
+And you're all set.
```

Comment:
In this case to be fully set is needed to define the folders where the flows and the frozen flows will be defined
```
smartbox_integration_camel_config:
    flows_directories:
    frozen_flows_directory:
```

Reply:
Fixed